fhttp/httprunner_test.go

```diff
@@ -57,8 +56,32 @@ func TestHTTPRunner(t *testing.T) {
 	}
 }
 
+func TestHTTPRunnerClientRace(t *testing.T) {
+	http.HandleFunc("/echo1/", EchoHandler)
+	port := DynamicHTTPServer(false)
+	URL := fmt.Sprintf("http://localhost:%d/echo1/", port)
+
+	opts := HTTPRunnerOptions{
```

Comment:
non-blocking: it might be nice to have a func just for setting qps options. maybe: `WithQPS(qps int32)` or similar?

Reply:
you're right - changing those to `opts.QPS = `